tap/src/main/webapp/capabilities.xml

```diff
@@ -1,193 +1,143 @@
+      <securityMethod standardID="ivo://ivoa.net/sso#OpenID"/>
     </interface>
-    <interface xsi:type="vs:ParamHTTP" role="std" version="1.1">
-      <accessURL use="base">https://replace.me.com/tap/async</accessURL>
-      <securityMethod standardID="ivo://ivoa.net/sso#tls-with-certificate"/>
-    </interface>
-    <!-- cannot use TAPRegExt-1.0 and xsi:type=tr:TableAccess because it restricts the
-        standardID to the plain TAP value -->
-  </capability>
 
-  <!-- TAP-1.0 base -->
-  <capability standardID="ivo://ivoa.net/std/TAP" 
-        xmlns:tr="http://www.ivoa.net/xml/TAPRegExt/v1.0" xsi:type="tr:TableAccess">
-    <interface xsi:type="vs:ParamHTTP" role="std">
-      <accessURL use="base">http://replace.me.com/tap</accessURL>
-    </interface>
-    <interface xsi:type="vs:ParamHTTP" role="std">
-      <accessURL use="base">https://replace.me.com/tap</accessURL>
-      <securityMethod standardID="ivo://ivoa.net/sso#tls-with-certificate"/>
-    </interface>
-    <!-- cannot describe the #BasicAA access because TAP-1.0 only has a base URL standardID -->
-    
     <!--
-    <dataModel ivo-id="ivo://ivoa.net/std/ObsCore/v1.0">ObsCore-1.0</dataModel>
+    dataModel ivo-id="ivo://ivoa.net/std/ObsCore/v1.0">ObsCore-1.0</dataModel>
     -->
-    
+
     <language>
-        <name>ADQL</name>
-        <version ivo-id="ivo://ivoa.net/std/ADQL#v2.0">2.0</version>
-        <description>ADQL-2.0</description>
-        <languageFeatures type="ivo://ivoa.net/std/TAPRegExt#features-adqlgeo">
-            <feature>
-                <form>POINT</form>
-            </feature>
-            <feature>
-                <form>CIRCLE</form>
-            </feature>
-            <feature>
-                <form>BOX</form>
-            </feature>
-            <feature>
-                <form>POLYGON</form>
-            </feature>
-            <feature>
-                <form>REGION</form>
-            </feature>
-            <feature>
-                <form>CONTAINS</form>
-            </feature>
-            <feature>
-                <form>INTERSECTS</form>
-            </feature>
-            <feature>
-                <form>AREA</form>
-            </feature>
-            <feature>
-                <form>CENTROID</form>
-            </feature>
-            <feature>
-                <form>COORDSYS</form>
-            </feature>
-            <feature>
-                <form>COORD1</form>
-            </feature>
-            <feature>
-                <form>COORD2</form>
-            </feature>
-        </languageFeatures>
+      <name>ADQL</name>
+      <version ivo-id="ivo://ivoa.net/std/ADQL#v2.0">2.0</version>
+      <description>ADQL-2.0</description>
+      <languageFeatures type="ivo://ivoa.net/std/TAPRegExt#features-adqlgeo">
+        <feature>
+          <form>POINT</form>
+        </feature>
+        <feature>
+          <form>CIRCLE</form>
+        </feature>
+        <feature>
+          <form>DISTANCE</form>

```

Comment:
I see `DISTANCE` was added here.  Was this verified to actually work in an ADQL query against the service?